clang/test/ARCMT/verify.m

```diff
@@ -11,7 +11,7 @@
 #error
 // expected-error@-1 {{}}
 
-//      CHECK: error: no expected directives found: consider use of 'expected-no-diagnostics'
+//      CHECK: error: no expected directives found: consider use of '{{.*}}-no-diagnostics'
```

Comment:
Do all these tests really fail if you leave "expected-no-diagnostics" in the expected output? Or can you just change that to a concrete value instead of the regex?

Reply:
yes since I changed the error output value in the tds file, these tests needed to be changed with the regex.

Reply:
And what is the output now? Both RUN lines only pass `-verify` so the prefix shouldn't change?

Reply:
Oh, I think I overlooked this. I then would have to revert the test changes. Please give me a confirmation to do so and also suggest whether or not I should write separate test cases for -verify=foo,bar for the change that this PR intends to implement.

Reply:
This test only passes `-verify` though, what's the new output and why doesn't the old expected output work?

Reply:
the intended behavior is that when only `-verify` is passed and there are no diagnostics in the file then `expected-no-diagnostics` should be the error and when `-verify=foo` is passed and no such diagnostics is present in the file, then `foo-no-diagnostics` should be the error.

Reply:
@tbaederr, I understand what you are trying to point out and I hope I rectified that in my new commit. I will add a screenshot of the working implementation also.